slave/buildslave/monkeypatches/bug4881.py

```diff
@@ -189,7 +191,7 @@ def _resourceFDImplementation(self):
         # to close
         if maxfds > 1024:
             maxfds = 1024
-        return xrange(maxfds)
+        return list(range(maxfds))
```

Comment:
Yeh, I think you have it backwards ... based on my understanding of that doc: 
  xrange should become range()
  range should become list(range())
That sound right?